.examples/discord_example.py

```diff
@@ -0,0 +1,20 @@
+# This example uses discord.py
+
+import discord
```

Comment:
Please add `discord.py` and `asyncio` to a `requirements.txt` file.

Reply:
You can't install asyncio

and also we don't have to install discord.py as it is just an example